Source/Scene/Camera.js

```diff
@@ -233,6 +235,13 @@ define([
         mag += mag * Camera.DEFAULT_VIEW_FACTOR;
         Cartesian3.normalize(this.position, this.position);
         Cartesian3.multiplyByScalar(this.position, mag, this.position);
+
+        /**
+         * If set, the camera will have its movement constrained.
```

Comment:
`the camera will have its movement constrained.` -> `the camera's position will be constrained.`